website/src/data/components/scatterplot/props.js

```diff
@@ -165,12 +165,24 @@ const props = [
 
             If you use a time scale, you must provide a time format
             as values are converted to Date objects.
-            
+
             Under the hood, nivo uses [d3-format](https://github.com/d3/d3-format),
             please have a look at it for available formats, you can also pass a function
             which will receive the raw value and should return the formatted one.
         `,
     },
+    {
+        key: 'nodeId',
+        group: 'Base',
+        defaultValue: defaults.nodeId,
+        type: 'string | (datum: NodeIdDatum): string',
+        help: `ID accessor for the node.`,
+        description: `
```

Comment:
I think it's worth mentioning that the node ID impacts the animations, nodes are gonna transition from a previous state at the same index, thus, it can be useful to manipulate this property if we want to establish some continuity between nodes at different index.